jekyll/_docs/test-metadata.md

```diff
@@ -7,8 +7,9 @@ description: Collecting test metadata
 
 CircleCI can collect test metadata from JUnit XML files and Cucumber JSON files.
 We'll use the test metadata to give you better insight into your build. For our
-inferred steps that use parallelism, we'll use the timing information to get you
-better test splits and finish your builds faster.
+inferred steps that use parallelism, we'll always use the timing information to get you
+better test splits and finish your builds faster. You can also take advantage
+of this runtime-based test splitting in custom steps if you follow [these steps](#using-the-files-modifier).
```

Comment:
This link `[these steps](#using-the-files-modifier)` needs to be changed to `[these steps]({{site.baseurl}}/test-metadata/#using-the-files-modifier)`. This is due to relative links failing on RSS feeds.